cmd/keyloader.go

```diff
@@ -25,7 +25,7 @@ import (
 	"github.com/testifysec/witness/options"
 )
 
-func loadSigners(ctx context.Context, ko options.KeyOptions) ([]cryptoutil.Signer, []error) {
+func loadSigners(ctx context.Context, ko options.KeyOptions, args []string) ([]cryptoutil.Signer, []error) {
```

Comment:
instead of passing the args here we should pass something more specific.

it seems to only ever be used in the call for a delegated signer, but i can't tell what this represents at all from this use.